### DISCUSSION TITLE
could not established to "sga.domcloud.co"

### DISCUSSION THREAD

**Discussion by bimmoismaya** [domcloud/domcloud#78] [General]
![Screenshot 2024-06-10 112606](https://github.com/domcloud/domcloud/assets/172015168/efbc8d96-9cc8-4de5-83c6-9da96956ddea)

**Comment by willnode** [comment]
Hello, it looks like your SSH connection logs out in some time. This is a known issue and in active investigation, but you should be able to relogin again (do reload window in vscode).

If you still not able to connect, probably you've entering password wrong for ten times. In that case you have to wait for 12 hours or change your connection. There cloud be your ISP blocking SSH connection at all.

You can chat with us (link in footer) to resolve this privately

**Reply by bimmoismaya** [reply]
should i do vscode update? coz vscode recommend to update app, but my laptop not support for install

**Reply by willnode** [reply]
When VSCode asks for what platform it uses... have you choose **Linux**? ![](https://code.visualstudio.com/assets/docs/remote/ssh/ssh-select-platform.png)

**Reply by bimmoismaya** [reply]
windows 😿 
after entering password vscode gimme notification like on title

**Reply by willnode** [reply]
You can reset the settings in `remote.SSH.remotePlatform` mentioned [here](https://code.visualstudio.com/docs/remote/ssh#_connect-to-a-remote-host).

**Reply by bimmoismaya** [reply]
i think my laptop get restricted by company, maybe i will test on another laptop, thanks for advice

**Reply by bimmoismaya** [reply]
![image](https://github.com/domcloud/domcloud/assets/172015168/fc84e2dc-7681-431b-b45c-7677db6e41c9)
by using linux on platform, all opened hahahaha, its so funny 😆
